src/components/Auth/Login.jsx

```diff
@@ -1,116 +1,99 @@
 
-  const {
-    state: { password, isEmailValid, email, display },
-    passwordDispatch
-  } = useAuth();
+    const handleEmailInputChange = (event) => {
+        credentialsDispatch({type: "EMAIL", payload: event.target.value})
+    }
 
-  const getClassName = (display) =>
-    display === "none" ? "pass-check-text display-n" : "pass-check-text";
+    const handlePasswordInputChange = (event) => {
+        credentialsDispatch({type: "PASSWORD", payload: event.target.value})
+    }
 
-  const getButtonState = (password) => {
-    return password.length < 6 ||
-      password === "" ||
-      !password.match("^[A-Za-z0-9]+$") || email === ""
-      ? true
-      : false;
-  };
+    const handlePasswordCheck = () => {
+        passwordtype === "password" ? setPasswordType("text") : setPasswordType("password")
+    }
 
-  return (
-    <div className="d-grid">
-      <div className="login-auth direction-column d-flex justify-center">
-        <h2 className="auth-title">Login</h2>
-        <div className="form-container">
-          <label className="form-label">Email address</label>
-          <input
-            type="email"
-            className="form-input lh-ls"
-            placeholder="name@example.com"
-            onChange={(e) =>
-              passwordDispatch({
-                type: "EMAIL_CHECK",
-                payload: e.target.value
-              })
-            }
-          />
-        </div>
-        {!isEmailValid && email !== "" && (
-          <p className="pass-check-text">
-            Email Id should be of type name@example.com
-          </p>
-        )}
+    const handleTestCredentialsLogin = () => {
+        userLogin("adarshbalika@gmail.com", "adarshbalika")
+    }
 
-        <div className="form-container relative">
-          <label className="form-label">Password</label>
-          <input
-            type={passwordtype}
-            className="form-input lh-ls"
-            placeholder="*********"
-            onChange={(e) =>
-              passwordDispatch({
-                type: "PASSWORD_CHECK",
-                payload: e.target.value
-              })
-            }
-          />
-          <button
-            class="button cursor"
-            onClick={() =>
-              passwordtype === "password"
-                ? setPasswordType("text")
-                : setPasswordType("password")
-            }
-          >
-            <span class="material-icons-outlined absolute pwd-icon-position">
-              visibility_off
-            </span>
-          </button>
-        </div>
-        <div className={getClassName(display)}>
-          <span>Password must contain atleast 6 characters</span>
-          <span>Password cannot contain any special character</span>
-        </div>
+    const handleLoginClick = () => {
+        if (credentials.email !== "" && credentials.password !== "") {
+            userLogin(credentials.email, credentials.password)
```

Comment:
try to use ternary operator to make it more readable

Reply:
I felt this was more readable so i wrote it this way, but thanks for the suggestion.